requirements.txt

```diff
@@ -9,4 +9,7 @@ fastapi==0.97.0
 uvicorn==0.22.0
 python-dotenv==1.0.0
 requests~=2.31.0
-https://github.com/explosion/spacy-models/releases/download/en_core_web_lg-3.5.0/en_core_web_lg-3.5.0.tar.gz
\ No newline at end of file
+https://github.com/explosion/spacy-models/releases/download/en_core_web_lg-3.5.0/en_core_web_lg-3.5.0.tar.gz
+langchain==0.0.209
+tiktoken==0.4.0
+faiss-cpu==1.7.4
```

Comment:
what does this library do?

Reply:
it's library to handle vectors and similarity search , tiktoken and fais-cpu are required for langchian library to work properly